cogs/commands.py

```diff
@@ -27,6 +29,9 @@ def _urlencode(*args, **kwargs):
 requests.models.urlencode = _urlencode
 
 
+# ################### COMMANDS ################### #
+
+
```

Comment:
I'm not totally sure how this got here. Must have happened during my rebase. Feel free to remove it.